vm.go

```diff
@@ -351,6 +353,51 @@ func (vm *VirtualMachineSpec) ExecuteCommand(command string) error {
 	return nil
 }
 
+// RunCommand executes a command on the Virtual Machine
+func (vm *VirtualMachineSpec) RunCommand(command string) (output string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Minute)
+	defer cancel()
+	system, err := hcs.OpenComputeSystem(ctx, vm.ID)
+	if err != nil {
+		return "", err
+	}
+	defer system.Close()
+
+	commandArray := strings.Split(command, " ")
```

Comment:
>command [](http://example.com/codeflow?start=31&length=7)

can we blindly split command with space?  What if the argument is quoted (either single (linux) or double (windows))?

Reply:
got it, change to passing as array of strings...